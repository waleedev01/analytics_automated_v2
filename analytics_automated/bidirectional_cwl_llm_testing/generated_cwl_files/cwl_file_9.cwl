cwlVersion: v1.0
class: CommandLineTool
baseCommand: echo
outputs:
  - id: output_file
    type: File
    outputBinding:
      glob: output.txt
inputs:
  - id: input_file
    type: File
    inputBinding:
      position: 1
requirements:
  DockerRequirement:
    dockerPull: ubuntu:latest