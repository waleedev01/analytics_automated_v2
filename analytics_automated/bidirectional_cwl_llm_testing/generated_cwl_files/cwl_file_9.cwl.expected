{"is_valid": false, "error": "Validation failed: Please define outputs in CWL as dictionary"}